mobile-app/app/screens/AppNavigator/screens/Loans/components/LoanCards.tsx

```diff
@@ -572,10 +574,9 @@ function LoanCard({
             </ThemedText>
           </View>
         )}
-        value={currentPrice}
+        value={getPrecisedCurrencyValue(currentPrice)}
```

Comment:
let's revert this change since it has been applied in line 550

Reply:
reverted